src/harshadBJun20/assignment28/SumOfAllNumbersInStringUsingAsciiValue.java

```diff
@@ -0,0 +1,35 @@
+package harshadBJun20.assignment28;
+
+/*Assignment 28 : 16th July 2020
+
+WAP to sum all the numbers in given string, without using Character class method. 
+(Note : Use Ascii value concept) .
+
+Input: J7u9y8h1h8j4b7j3j6
+Ouput: Sum of all numbers in String is - 53.*/
+
+public class SumOfAllNumbersInStringUsingAsciiValue {
+	
+	void calSumOfAllNumbers(String str) {
+		int sumOfDigit=0;
+		String digitValue="";
+		for(int index=0;index<str.length();index++) {
+			if(str.charAt(index)==48||str.charAt(index)==49||str.charAt(index)==50||str.charAt(index)==51||str.charAt(index)==52||str.charAt(index)==53||str.charAt(index)==54||str.charAt(index)==55||str.charAt(index)==56||str.charAt(index)==57) {
```

Comment:
When requirement is clearly mentioned like WAP to sum all the numbers in given string, **without using Character class method.**  then why you have used it.